packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuReactRootViewContainer.kt

```diff
@@ -2,11 +2,29 @@ package expo.modules.devmenu
 
 import android.content.Context
 import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
 import android.widget.FrameLayout
+import com.facebook.react.ReactRootView
+import expo.modules.devmenu.fab.MovableFloatingActionButton
+
+/**
+ * We want to hide the FAB behind the feature flag for now.
+ */
+private const val enableFAB = false
 
 class DevMenuReactRootViewContainer(context: Context) : FrameLayout(context) {
+  private val fab = MovableFloatingActionButton(context)
```

Comment:
```suggestion
  private val fab by lazy { MovableFloatingActionButton(context) }
```

nit: in case feature flag is not enabled